mods/chat.js

```diff
@@ -1,573 +1,603 @@
+    ranks: ['', 'banned', 'user', 'mod', 'owner'],
+    applyMembers: function (data) {
         var applyMemberData = function () {
-            var addMemberElement = function ( chat, chatId, userId ) {
+            var addMemberElement = function (chat, chatId, userId) {
                 var usedPlatforms = '';
                 for (var pkey in idrinth.chat.users[userId].platforms) {
-                    if ( idrinth.chat.users[userId].platforms[pkey] ) {
+                    if (idrinth.chat.users[userId].platforms[pkey]) {
                         usedPlatforms += pkey;
                     }
                 }
-                chat.appendChild ( idrinth.ui.buildElement (
-                        {
-                            type: 'li',
-                            css: 'user ' + idrinth.chat.ranks[parseInt ( idrinth.chat.chatRank[chatId][userId], 10 )],
-                            content: ( usedPlatforms === '' ? '' : '[' + usedPlatforms.toUpperCase () + '] ' ) + idrinth.chat.users[userId].name,
-                            attributes:
-                                    [
-                                        { name: 'data-id', value: userId },
-                                        { name: 'onclick', value: 'idrinth.chat.userclick(this,' + userId + ', ' + chatId + ')' }
-                                    ]
-                        }
-                ) );
+                chat.appendChild(idrinth.ui.buildElement({
+                        type: 'li',
+                        css: 'user ' + idrinth.chat.ranks[parseInt(idrinth.chat.chatRank[chatId][userId], 10)],
+                        content: ( usedPlatforms === '' ? '' : '[' + usedPlatforms.toUpperCase() + '] ' ) + idrinth.chat.users[userId].name,
+                        attributes: [
+                            {name: 'data-id', value: userId},
+                            {name: 'onclick', value: 'idrinth.chat.userclick(this,' + userId + ', ' + chatId + ')'}
+                        ]
+                    }
+                ));
             };
             for (var chatId in idrinth.chat.chatRank) {
-                if ( document.getElementById ( 'idrinth-chat-tab-' + chatId ) ) {
-                    var chat = document.getElementById ( 'idrinth-chat-tab-' + chatId ).getElementsByTagName ( 'ul' )[0];
-                    while ( chat.firstChild ) {
-                        chat.removeChild ( chat.firstChild );
+                if (document.getElementById('idrinth-chat-tab-' + chatId)) {
+                    var chat = document.getElementById('idrinth-chat-tab-' + chatId).getElementsByTagName('ul')[0];
+                    while (chat.firstChild) {
+                        chat.removeChild(chat.firstChild);
                     }
                     for (var userId in idrinth.chat.chatRank[chatId]) {
-                        addMemberElement ( chat, chatId, userId );
+                        if (!idrinth.chat.chatRank[chatId].hasOwnProperty(userId)) continue;

```

Comment:
do not skip brackets please, this is too likely to create errors in the future.
Also don't use continues, replace them with a function call with early return